content/docs/user-guide/basic-concepts/experiment.md

```diff
@@ -7,5 +7,5 @@ tooltip: >-
   experiments](/doc/start/experiments), having [built-in
   mechanisms](/doc/user-guide/experiment-management) like the
   [run-cache](/doc/user-guide/project-structure/internal-files#run-cache) and
-  the `dvc experiments` commands (coming in DVC 2.0).
+  the `dvc experiments` commands (This feature is only available in DVC 2.0).
```

Comment:
💅 This -> this

Reply:
this feature is available in DVC 2.0+

(it's not "only" - there should be some good phrase to communicate 2.0+ - I would check other tools)